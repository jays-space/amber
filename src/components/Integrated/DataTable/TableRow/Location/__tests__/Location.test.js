import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Location } from "..";

describe("<Location />", () => {
  const mockLocationData = {
    city: "johannesburg",
    country: "south africa",
  };

  it("should render correctly", () => {
    render(
      <Location
        city={mockLocationData.city}
        country={mockLocationData.country}
      />
    );

    const locationComponent = screen.getByTitle("candidate-city-country");
    expect(locationComponent).toBeInTheDocument();
    expect(locationComponent).toMatchSnapshot();
  });
});
