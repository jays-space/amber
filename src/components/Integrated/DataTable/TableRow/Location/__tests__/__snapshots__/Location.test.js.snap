// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Location /> should render correctly 1`] = `
<div
  title="candidate-city-country"
>
  <span>
    <p
      class="tracking-wide  text-cyan-900 font-semibold w-10/12 text-lg whitespace-nowrap text-ellipsis overflow-hidden capitalize"
      data-testid="paragraph"
    >
      johannesburg
    </p>
  </span>
  <span>
    <p
      class="tracking-wide  text-slate-700 capitalize opacity-75"
      data-testid="paragraph"
    >
      south africa
    </p>
  </span>
</div>
`;
